packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -164,13 +165,29 @@ A pie chart may be the wrong choice when you need to compare categories to one a
 1. **Pie chart fill:** We recommend that fill colors should be based on the [Colors for charts](/design-guidelines/styles/chart-colors).
 2. **Legend:** Each variable on the legend should report their current value.
 
-### React component
+#### React component
 [Pie chart](/documentation/react/charts/chartpie)
 
+### Sparkline
+
+A sparkline is a small chart that helps users to analyze data and understand trends and patterns.
+
+#### Usage
+Sparklines are commonly used in tables, reports, and dashboards. A common use case is displaying utilization in an environment over time. Sparklines help to show trends in a series of values or to highlight maximum and minimum values. We recommend using sparklines alone without an axis and placing the name of the item being visualized underneath the chart. 
+
+#### Example
+![Sparkline](./img/sparkline.png)
+
+1. **Axis:** We recommend that sparklines are shown without an x or y-axis. Depending on space, a label can be placed underneath or to the left of the sparkline.
+
+### React component
```

Comment:
Looks like you missed this one :(

Reply:
Ah shoot, just pushed (hopefully) the last change!